WordPress/src/main/java/org/wordpress/android/util/WPWebViewClient.java

```diff
@@ -45,6 +42,12 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (!url.equals("file:///android_asset/webkit/")) {
             view.loadUrl(url);
         }
+        // Intercept requests for private images and add the WP.com authorization header
+        if (mBlog != null && mBlog.isPrivate() && !TextUtils.isEmpty(mToken)) {
+            Map<String, String> headers = new HashMap<>();
+            headers.put("Authorization", "Bearer " + mToken);
```

Comment:
Do we know whether these headers will be provided to image requests resulting from this url?